apps/verticalhome/manifest.webapp

```diff
@@ -1,55 +1,69 @@
-{
-  "name": "Homescreen",
-  "description": "Gaia Homescreen",
-  "launch_path": "/index.html",
-  "type": "certified",
-  "role": "homescreen",
-  "developer": {
-    "name": "The Gaia Team",
-    "url": "https://github.com/mozilla-b2g/gaia"
-  },
-  "permissions": {
-    "themeable":{},
-    "mobileconnection":{},
-    "open-remote-window":{},
-    "storage": {},
-    "webapps-manage": {},
-    "systemXHR": {},
-    "settings":{ "access": "readwrite" }
-  },
-  "default_locale": "en-US",
-  "locales": {
-    "ar": {
-      "name": "الشاشة الرئيسية",
-      "description": "Gaia الشاشة الرئيسية"
-    },
-    "en-US": {
-      "name": "Homescreen",
-      "description": "Gaia Homescreen"
-    },
-    "fr": {
-      "name": "Écran d’accueil",
-      "description": "Écran d’accueil Gaia"
-    },
-    "zh-TW": {
-      "name": "主畫面",
-      "description": "Gaia 主畫面"
-    }
-  },
-  "icons": {
-    "84": "/style/icons/vertical_84.png",
-    "126": "/style/icons/vertical_126.png",
-    "142": "/style/icons/vertical_142.png",
-    "189": "/style/icons/vertical_189.png",
-    "284": "/style/icons/vertical_284.png"
-  },
-  "orientation": "default",
-  "activities": {},
-  "messages": [],
-  "datastores-owned": {
-    "vertical_preferences_store": {
-      "access": "readwrite",
-      "description": "Stores preferences"
-    }
-  }
-}
+{
+  "name": "Homescreen",

```

Comment:
it seems that in this file line ending is wrong.
Could you check it and if it is so, correct it?